sdcm/utils/remote_logger.py

```diff
@@ -299,6 +300,38 @@ def _logger_cmd(self) -> str:
         return f"{cmd} >> {self._target_log_file} 2>&1"
 
 
+class WrongSchedulingLogger(CommandClusterLoggerBase):
```

Comment:
I think it must include some K8S-related word in it's name.

Reply:
changed to `KubernetesWrongSchedulingLogger`